### PR TITLE
CYB-503 guide to upgrade daobet node from 1.x to 2.0

### PR DESCRIPTION
**Pull request by adrianopol**

### CODE REVIEW

File: docs/upgrade-1.x-to-2.0.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  ```
+
+* Create file `protocol_features/BUILTIN-PREACTIVATE_FEATURE.json` in your config directory with the following contents:
+  ```
+  {
+    "protocol_feature_type": "builtin",
+    "dependencies": [],
+    "description_digest": "64fe7df32e9b86be2b296b3f81dfd527f84e82b98e363bc97e40bc7a83733310",
+    "subjective_restrictions": {
+      "earliest_allowed_activation_time": "1970-01-01T00:00:00.000",
```
Comment (algys) [inline docs/upgrade-1.x-to-2.0.md:36]: mb set real time ? like `2020-04-23`, it's just example, need to discuss about particular date ?

File: docs/upgrade-1.x-to-2.0.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  docker pull daocasino/daobet:v2.0.0-rc2
+  ```
+
+* Don't forget to enable block production either by setting `pause-on-startup` config option to `false` (before node
+  run), or by performing a HTTP GET-request to http://localhost:8888/v1/producer/resume (after node run).
+
+* Run node (verson 2.0.0).
+
+* ```
+  apt-get install jq && curl -X POST http://127.0.0.1:8888/v1/producer/schedule_protocol_feature_activations \
```
Comment (algys) [inline docs/upgrade-1.x-to-2.0.md:74]: we need to make this action after at least 2/3+1 validators update their nodes, so we need to get time for validators to update and set deadline by `earliest_allowed_activation_time` to ensure it

File: docs/upgrade-1.x-to-2.0.md (hunk 1 above)
Comment (adrianopol) [reply docs/upgrade-1.x-to-2.0.md:36]: @algys, this date should be chosen on BPs' quorum. Date in the file is just an example for understanding format of the value.

File: docs/upgrade-1.x-to-2.0.md (hunk 2 above)
Comment (adrianopol) [reply docs/upgrade-1.x-to-2.0.md:74]: Oops! Duplication, thank you.

Comment (adrianopol) [reply docs/upgrade-1.x-to-2.0.md]: Removed curl call duplication.

File: docs/upgrade-1.x-to-2.0.md (hunk 1 above)
Comment (algys) [reply docs/upgrade-1.x-to-2.0.md:36]: Ok, but I guess we should add text about it(that before updating BPs need to choose that date) to instruction

Comment (adrianopol) [reply docs/upgrade-1.x-to-2.0.md:36]: See line 44.

**Pull request merged**